lib/src/widgets/views/build_pagination.dart

```diff
@@ -87,11 +87,11 @@ class BuildPagination<T> extends StatelessWidget {
           physics: physics,
           shrinkWrap: shrinkWrap,
           padding: padding,
-          itemCount: items.length + (isLoading ? 1 : 0),
+          itemCount: 1 + (isLoading ? 1 : 0),
```

Comment:
This shouldn't be changed, else the `index` will max out at 1 or 2.

Reply:
Yes, so developers will have access to whole list entire time and can use index to select the specific data from that list if required.

This will also give developers the no. of documents loaded directly.